ReactWindows/ReactNative.Shared/Modules/Dialog/DialogModuleHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ReactNative.Modules.Dialog
+{
+    static class DialogModuleHelper
```

Comment:
I was hoping to leave room for eventual shared code here, perhaps shared methods.  Sort of a placeholding class for constants and helper methods.  I can extract them if you'd like.  Let me know what you prefer.

The readability seems good.